semanticdb-kotlinc/src/main/kotlin/com/sourcegraph/semanticdb_kotlinc/SemanticdbTextDocumentBuilder.kt

```diff
@@ -133,6 +133,7 @@ class SemanticdbTextDocumentBuilder(
         if (kdoc.isEmpty()) return kdoc
         val out = StringBuilder().append("\n\n").append("----").append("\n")
         kdoc.lineSequence().forEach { line ->
+            if (line.isEmpty()) return ""
```

Comment:
Is this strictly equivalent to the proposed fix? 

```kotlin
            if (line.length <= 0) { continue } // Suggested adding this check
```

This to me reads "skip empty lines, but eventually process every line"

The version in this PR reads "the moment you encounter an empty line anywhere in the doc, terminate the entire function with an empty string"

Unless in Kotlin return + forEach have some different semantics

Reply:
> The version in this PR reads "the moment you encounter an empty line anywhere in the doc, terminate the entire function with an empty string"

Yup youre absolutely right, I messed up the test while I was checking semantics of kdoc comments (line prefixed with `*` or not etc) and forgot actually include the exception case :face_exhaling:

Reply:
Fixed https://github.com/sourcegraph/lsif-kotlin/pull/54/commits/4d77f55201a735192379cbb1a93192b797c82939